src/components/add-boiler-house-form/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { axiosInstance } from "@mtfh/common";
+import { Asset } from "@mtfh/common/lib/api/asset/v1";
+import { config } from "@mtfh/common/lib/config";
+
+export interface PatchAssetRequest {
+  boilerHouseId: string;
+}
+
+export const patchAsset = async (
```

Comment:
Isn't this call set up in Common MFE?

Reply:
Yeah, I must have forgotten to change the import